packages/libxfce4util.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Libxfce4util < Package
+    description 'Utility library for the Xfce4 desktop environment'
+    homepage "https://xfce.org/"
+    version '4.14'
+    compatibility 'all'
+    source_url "https://archive.xfce.org/src/xfce/libxfce4util/4.14/libxfce4util-#{version}.0.tar.bz2"
```

Comment:
I prefer to not use `#{version}` in urls.  It makes it easier to `sha256sum` and cut and paste.